### PR TITLE
Shadow Shaman, Warlock and Witch Doctor Ultimate buffs.

### PR DESCRIPTION
**Pull request by POCKETMONEY99**
Buffed all the summoned units from Shadow Shaman, Warlock and WD ultimates. 
Increased the damage that Shadow Shaman Serpent Wards do. 
Increased the durability of Warlock Golems substaintially and increased their damage output. 
Increased the damage that Witchdoctor ultimate does from levels 1 to 4. I nerfed the damage on the 25 talent from 600 to 400. This hopefully makes withdoctors ultimate good without the talent.

### CODE REVIEW

**Comment by chrisinajar** [comment]
WIP?

File: game/scripts/npc/abilities/warlock_rain_of_chaos.txt [hunk 1]
```diff
@@ -58,24 +58,23 @@
...
       {
         "var_type"                                        "FIELD_INTEGER"
-        "golem_hp_tooltip"                                "1000 2000 3000 5000 10000"
+        "golem_hp_tooltip"                                "1000 3000 5000 7500 10000" //OAA
       }
       "05"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "golem_dmg_tooltip"                               "100 150 200 400 700"
-        "CalculateSpellDamageTooltip"                     "0"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/warlock_rain_of_chaos.txt]: this

File: game/scripts/npc/abilities/warlock_rain_of_chaos.txt [hunk 2]
```diff
@@ -86,13 +85,12 @@
       "09" // 75% of golem_hp_tooltip
       {
         "var_type"                                        "FIELD_INTEGER"
-        "golem_hp_tooltip_scepter"                        "750 1500 2250 3750 7500"
-        "RequiresScepter"                                 "1"
```
Comment (DarkoniusXNG) [inline game/scripts/npc/abilities/warlock_rain_of_chaos.txt]: this

**Comment by POCKETMONEY99** [comment]
> 
> 
> WIP?

**Comment by POCKETMONEY99** [comment]
> 
> 
> WIP?

Finished it. I took a very long sleep deprived hitatus but it should work now.

**Pull request merged**